doc/api/util.md

```diff
@@ -68,8 +68,7 @@ exports.puts = util.deprecate(() => {
 It returns a modified function which warns once by default.
 
 If `--no-deprecation` is set then this function is a NO-OP.  Configurable
-at run-time through the `process.noDeprecation` boolean (only effective
-when set before a module is loaded.)
+at run-time through the `process.noDeprecation` boolean.
```

Comment:
I think `boolean` is fine. If you’d like to see this changed, then I’d prefer to extend it to `boolean property` and apply the same change to the paragraphs directly below this one.